doc/manuals/admin/cli.md

```diff
@@ -86,7 +86,12 @@ The list of available options is the following:
 -   **-logAppend**. If used, the log lines are appended to the
     contextBroker log file, instead of re-creating it when the
     broker starts.
--   **--silent**. Suppress all log output except errors.
```

Comment:
What happen if --silent and -logLevel are used at the same time? In that case, two options:

* Exit in the same way we exit for concurrent usage of -t and -logLevel != DEBUG
* -logLevel sinlently overrides --silent